03-pipefilter.md

```diff
@@ -516,3 +516,19 @@ so this matches all the valid data files she has.
 > What other command(s) could be added to this in a pipeline to find
 > out what animals the file contains (without any duplicates in their
 > names)?
+
+> ## Understanding the commands `head`, `tail` and `>>` {.challenge}
+>
+> Consider the file `animals.txt`, used in previous exercise. 
+> After these following commands, select the alternative that 
+> corresponds the file `animalsUpd.txt`:
+>
+> ~~~ {.bash}
```

Comment:
What you think of

~~~
> After run
>
> ~~~ {.bash}
> $ head -3 animals.txt > animalsUpd.txt
> $ tail -2 animals.txt >> animalsUpd.txt
> ~~~
>
> what the following alternatives best describe the content of `animalsUpd.txt`?
~~~

You don't need to make this change since this is only cover writing style.